nipype/external/six.py

```diff
@@ -186,8 +187,8 @@ def find_module(self, fullname, path=None):
     def __get_module(self, fullname):
         try:
             return self.known_modules[fullname]
-        except KeyError:
-            raise ImportError("This loader does not know module " + fullname)
+        except KeyError as e:
+            raise_from(ImportError("This loader does not know module " + fullname), e)
```

Comment:
I see. Are all the files in ``external`` this way? If possible, we should avoid this